Formula/rna-star.rb

```diff
@@ -0,0 +1,35 @@
+class RnaStar < Formula
+  desc "RNA-seq aligner"
+  homepage "https://github.com/alexdobin/STAR"
+  # cite "https://doi.org/10.1093/bioinformatics/bts635"
```

Comment:
```suggestion
  # cite Dobin_2012: "https://doi.org/10.1093/bioinformatics/bts635"
```
Check out…
```sh
brew cite --ruby 10.1093/bioinformatics/bts635
```

Reply:
"Unknown command: cite". where to get the cite command?

Reply:
`brew tap brewsci/base`